### ISSUE TITLE
DMCA

### ISSUE THREAD

**Issue by chisaipete** [chisaipete/bestiary#86]
Hey folks, 

I've seen too many sites which are _questionably_ legal under US Copyright law get served DMCA's by WOTC Lawyers in recent years.  I feel like this site is too grey for me personally to continue to host on my personal github, in fear of ramifications and a clear conscience.  This isn't a place to argue or discuss the legality or the viability of the site, I'm just not comfortable with hosting it any longer.  That's my opinion and how it's going to play out.  Whatever you choose to do otherwise with forks and so-forth is none of my business.

In order to facilitate any kind of transition and to be kind to the community, I'll be shutting down and deleting this repository in 1 week.  Please contact me directly if there is something you need.

This has been a hard decision, but ultimately, I value content creators and wish to make sure they are compensated fairly for their content.  Rewarding creators is what I want to achieve, and I feel like this is something I can do to help that happen.

However, if someone can keep making this site work while using a paid service (i.e. dndbeyond) as a backend and keep only the SRD as free content, I'm all for continuing to host.  I support Wizards with DnDBeyond myself, but I'm assuming that at least some portion of the folks who use the content here are not purchasing this content otherwise.

Thank you for your support and contributions, 

--chisaipete

**Comment by Kyrremann** [comment]
I'm a bit surprised, but I understand that you want to avoid the DMCA. Personally I was pretty sure this whole site was illegal from the get go, and that we mostly just wanted to help people who couldn't afford all the books. Anyway, if you want, we could remove all the non SRD content, but then again, people could also just head over to http://5e.d20srd.org/indexes/monsters.htm or similar sites.

Maybe the easiest thing is to close the site down, and be happy with what you created? Forks will still be out there, so you could always change your mind.

I also support Wotc with buying physical and digital books, even though they are pretty pricy.

**Comment by chisaipete** [comment]
Yeah, I'm kind of torn, especially with the fantastic contributions that you've made recently @Kyrremann . I feel like the site really offers something niche...an actually fast, pleasantly and minimally styled monster listing.  I just had an epiphany of sorts and realized that hosting this threw something out of whack for me.  I don't actually use this bestiary as much as I do DnDBeyond these days.

**Comment by Kyrremann** [comment]
It's the same with me, I also mostly use DnDBeyond, but sometimes I look up some creatures here if I haven't bought the book yet.

It was a good run, and it was fun (for me) to contribute and get to work some more with Jekyll.
